samples/local_attestation/enclave_a/Makefile

```diff
@@ -36,7 +36,7 @@ build:
 	oeedger8r ../localattestation.edl --trusted --trusted-dir ../common \
 		--search-path $(INCDIR) \
 		--search-path $(INCDIR)/openenclave/edl/sgx
-	$(CXX) -g -c $(CXXFLAGS) $(INCLUDES) -I. -I.. -std=c++11 -DOE_API_VERSION=2 ecalls.cpp ../common/attestation.cpp ../common/crypto.cpp ../common/dispatcher.cpp
+	$(CXX) -g -c $(CXXFLAGS) $(INCLUDES) -I../../../include -I. -I.. -std=c++11 -DOE_API_VERSION=2 ecalls.cpp ../common/attestation.cpp ../common/crypto.cpp ../common/dispatcher.cpp
```

Comment:
We will need to test and verify against most recent OE to see if we can remove -I../../../include in the makefile.

Reply:
As of most recent OE, evidence.h is still not installed in oe/attestation/sgx/evidence.h by default.